### PR TITLE
Prepare for v0.0.18 release.

### PR DESCRIPTION
**Pull request by terrylucas**

### CODE REVIEW

File: packages/devtools/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
-## 0.0.17-dev.2 - 2019-04-29
+## 0.0.18 - 2019-04-30
 * Fix release bug (0.0.17-dev.1 did not include build folder)
```
Comment (kenzieschmoll) [inline packages/devtools/CHANGELOG.md]: wdyt about deleting this line? seems irrelevant now that the dev release headings have been removed.

File: packages/devtools/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-
-## 0.0.17-dev.1 - 2019-04-26
 * Add CPU profiler (preview) to timeline page.
+* CPU flame chart UI improvement and bug fixes.
```
Comment (kenzieschmoll) [inline packages/devtools/CHANGELOG.md]: improvement --> improvements

File: packages/devtools/CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+* Fixed DevTools working correctly in Chrome on Windows.
+* DevTools server released and support to launch Chrome automatically.
```
Comment (kenzieschmoll) [inline packages/devtools/CHANGELOG.md]: nit: wording - DevTools server released with support for launching DevTools in Chrome.

File: packages/devtools/CHANGELOG.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
-## 0.0.17-dev.2 - 2019-04-29
+## 0.0.18 - 2019-04-30
 * Fix release bug (0.0.17-dev.1 did not include build folder)
-
-## 0.0.17-dev.1 - 2019-04-26
 * Add CPU profiler (preview) to timeline page.
+* CPU flame chart UI improvement and bug fixes.
+* Fixed DevTools working correctly in Chrome on Windows.
```
Comment (kenzieschmoll) [inline packages/devtools/CHANGELOG.md]: nit: wording - Bug fixes for DevTools on Windows.

File: packages/devtools/CHANGELOG.md (hunk 1 above)
Comment (terrylucas) [reply packages/devtools/CHANGELOG.md]: In Dart once we move to a alpha/stable we combine the interim devs to the new major release.

File: packages/devtools/CHANGELOG.md (hunk 2 above)
Comment (terrylucas) [reply packages/devtools/CHANGELOG.md]: Done.

File: packages/devtools/CHANGELOG.md (hunk 3 above)
Comment (terrylucas) [reply packages/devtools/CHANGELOG.md]: Done.

File: packages/devtools/CHANGELOG.md (hunk 4 above)
Comment (terrylucas) [reply packages/devtools/CHANGELOG.md]: Done.

**Comment by terrylucas** [comment]
Thanks Kenzie.
